site/src/components/hero.md

```diff
@@ -114,6 +114,29 @@ On pages with right to left direction, the position of the hero image will remai
 </section>
 ```
 
+Hero images with details
+
```

Comment:
Can we add some text describing what it is?

Reply:
Adjusted.

Reply:
```suggestion
Hero images with details

This component variant overlays content details on hero images and aligns the content area to the right. It can be used to accommodate more content within the hero section if necessary.
```

Reply:
Committing suggestions doesn't seem to work well with GitHub. I still see the old text. Can you review and make the changes manually?

Reply:
Pushing up an edit on this.  Looks like somehow got duplicated text, anyways  should be cleaned up now.